edi_oca/tests/test_backend_process.py

```diff
@@ -58,7 +58,6 @@ def test_process_record_with_error(self):
             [
                 {
                     "edi_exchange_state": "input_processed_error",
-                    "exchange_error": "OOPS! Something went wrong :(",
```

Comment:
Can you try to verify if te message is inside it :smile: ?

Reply:
Sure!